examples/angular/src/pages/ui/components/authenticator/sign-in-federated/sign-in-federated.component.html

```diff
@@ -1,4 +1,6 @@
-<amplify-authenticator>
+<amplify-authenticator
+  [socialProviders]="['amazon', 'apple', 'facebook', 'google']"
```

Comment:
nit: looks like these 4 social providers are used at several different places, should we make a constant for them?

Reply:
Good point, but imo no, because they are getting re-used between different frameworks (`examples/vue`, `examples/angular`, ...). I think it makes sense to keep them isolated for now.